apps/dsp-app/cypress/e2e/system-admin/data-model.cy.ts

```diff
@@ -0,0 +1,61 @@
+import { faker } from '@faker-js/faker';
+import ProjectPage from '../../support/pages/project-page';
+
+describe('Data Model', () => {
+  const projectPage = new ProjectPage();
+
+  beforeEach(() => {
+    projectPage.requestProject();
+  });
+
+  it('should create new data model', () => {
+    const data = {
+      name: faker.string.alpha({ length: { min: 3, max: 16 } }),
+      label: faker.lorem.text(),
+      comment: faker.lorem.sentence(),
+    };
+
+    cy.intercept('POST', '/v2/ontologies').as('submitRequest');
+    cy.visit(`/project/${projectPage.projectUuid}/data-models`);
+    cy.url().should('match', new RegExp(`project\/${projectPage.projectUuid}\/data-models`));
+
+    cy.get('[data-cy=data-models-container]')
+      .find('[data-cy=create-button]')
+      .scrollIntoView()
+      .should('be.visible')
+      .click();
+
+    cy.get('[data-cy=name-input]').type(data.name).wait(200);
```

Comment:
why do you add wait() ? it is a bad pattern

Reply:
https://github.com/dasch-swiss/dsp-das/pull/1475#issuecomment-1959179537